spec/support/test_helpers/candidate_submission_helper.rb

```diff
@@ -0,0 +1,46 @@
+module CandidateSubmissionHelper
```

Comment:
I think we need to continue discussing how we improve these helper modules. We need to be clear about what goes in them and what doesn't. I'm not sure adding this here now is helping us with that.

Reply:
I just need to reuse many of the steps and I think adding to CandidateHelper will just make a module where we put everything, hence why I create this module.